### PR TITLE
Include weekly development meetings within documentation

### PR DESCRIPTION
**Pull request by chriscoffee**
**What is the problem I am trying to address?**

> The Athens development meeting happens weekly but nobody knows where to
> find it.. 

**How is the fix applied?**

I felt that the current organisation of these pages wasn't
easily readable so have sorted the community pages separately from
those on say using Git or GitHub. I took a lot of the content from the
google doc so I altered it slightly so it fits with the voice of the
current document.

The new structure is:
Contributing > The Athens Community > Participating in the Community
Contributing > The Athens Community > Developer Meetings

**Fixes issue:  #[730]**

Signed-off-by: Chris M <millscj01@gmail.com>

### CODE REVIEW

File: docs/content/contributing/community/developer-meetings.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Developer Meetings"
+date: 2018-09-28T10:40:50-07:00
+weight: 2
+
+---
+
+We hold our weekly developer meetings every Thursday at 11am US Pacific Time (UTC-7). They start with a quick update on the project from the Maintainers, and then we discuss things on the agenda.
```
Comment (robjloranger) [inline docs/content/contributing/community/developer-meetings.md]: I think we should consider leaving the timezone out as we would have to change it with daylight savings twice a year.

File: docs/content/contributing/community/developer-meetings.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Absolutely everyone is invited to attend, suggest topics for the agenda, and participate! You don’t have to be a contributor or maintainer to show up and share your perspective, ideas, feelings, etc.
+
+### Meeting Details
+
+They're every Thursday at 11am to 12pm US Pacific Time (UTC-7) and you can join the meeting here by following this link: https://zoom.us/j/193797787
```
Comment (robjloranger) [inline docs/content/contributing/community/developer-meetings.md]: as above

File: docs/content/contributing/community/developer-meetings.md (hunk 1 above)
Comment (carolynvs) [reply docs/content/contributing/community/developer-meetings.md]: Agreed

Comment (carolynvs) [reply docs/content/contributing/community/developer-meetings.md]: nit: Change "Maintainers" to "maintainers"

Comment (chriscoffee) [reply docs/content/contributing/community/developer-meetings.md]: Thanks, I've made a change to remove UTC-7 as suggested. 👍

File: docs/content/contributing/community/developer-meetings.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Here's an example of how to suggest a topic:
+
+> * (Bob) Let’s talk about something cool!
+
+> * (Alice) Let’s talk about cool thing #2!
+
+### Past Meetings
+
+Don't worry if you can't make them but want to catch up! All past development meetings available on [YouTube](https://www.youtube.com/playlist?list=PLAk08AWjk5sekD-FRjU4VVe97nltUyZ4W) for you to watch back anytime.
```
Comment (robjloranger) [inline docs/content/contributing/community/developer-meetings.md]: I might suggest rewording this last sentence a bit, but not huge. `..to watch back anytime.` sounds a bit strange to me.

Comment (chriscoffee) [reply docs/content/contributing/community/developer-meetings.md]: I took your point into account. You're right, it's weird. I'm blaming like needing sleep or something haha.

**Comment by arschles** [comment]
This closed #730. I'll close the issue accordingly
